app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError, NotWearingMaskError
+
+
+def go_to_cafe(self, friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+            return f"Friends can go to {cafe.name}"
```

Comment:
Returning immediately after the first successful visit may not be the intended behavior if multiple friends are visiting. Consider accumulating results or checking all friends before returning a message.